contracts/Cross-chain/BaseOmnichainControllerSrc.sol

```diff
@@ -43,7 +43,8 @@ contract BaseOmnichainControllerSrc is Ownable, Pausable {
      * @notice Emitted when the maximum daily limit of commands from the local chain is modified
      */
     event SetMaxDailyLimit(uint16 indexed chainId, uint256 oldMaxLimit, uint256 newMaxLimit);
-    /*
+
+    /**
```

Comment:
Is this change fixing linter? Or could we remove it?

Reply:
It was fixing the format of the natspec yes. Not necessarily brought by the linter